### DISCUSSION TITLE
Disconnecting from a server results in baro freezing for 10 seconds before reaching main menu

### DISCUSSION THREAD

**Discussion by Tsunder** [FakeFishGames/Barotrauma#13624] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

pressing "disconnect" button when connected to a server, just in the lobby, results in baro freezing for ~10s before reaching main menu.

the server isn't even in a round!

### Reproduction steps

1. join a MP server
2. press disconnect button

https://user-images.githubusercontent.com/5103517/168233227-975fdd99-c108-4b75-84ee-a23f873a161c.mp4

### Bug prevalence

Happens every time I play

### Version

0.17.15.0

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Could not reproduce

**Comment by Tsunder** [comment]
I believe this may have been addressed/fixed in some commit, but do not recall any specifics.
